brave/brave-core
# Refactored and migrated Rewards category from category to type

Fixes https://github.com/brave/brave-browser/issues/6350

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:

- Confirm Contributions state works as expected after an upgrade
- Confirm Pending Contributions state works as expected after an upgrade
- Confirm Contributions state works as expected for a fresh install
- Confirm Pending Contributions state works as expected for a fresh install

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
